utils/release-checklist

```diff
@@ -1,37 +1,112 @@
 Release checklist
 =================
 
+for stable releases
+-------------------
+
 * open https://travis-ci.org/sphinx-doc/sphinx/branches and check **X.Y** branch is green
 * Run `git status` and check nothing changed
-* Update sphinx/locale/sphinx.pot if first major release (beta1)
+* `python utils/bump_version.py X.Y.Z`
+* Check diff by `git diff`
+* Edit CHANGES if empty section exists
+* `git commit -am 'Bump to X.Y.Z final'`
+* `make clean`
+* `python setup.py release bdist_wheel sdist upload --identity=[your key]`
+* open https://pypi.python.org/pypi/Sphinx and check there are no obvious errors
+* `git tag vX.Y.Z`
+* `python utils/bump_version.py --in-develop X.Y.Zb0` (ex. 1.5.3b0)
+* Check diff by `git diff`
+* `git commit -am 'Bump version'`
+* `git push origin X.Y --tags`
+* `git checkout master`
+* `git merge X.Y`
+* `git push origin master`
+* Update `sphinx-doc-translations <https://github.com/sphinx-doc/sphinx-doc-translations>`_
+* Add new version/milestone to tracker categories
+* Write announcement and send to sphinx-dev, sphinx-users and python-announce
 
-  * Run `python setup.py extract_messages`
-  * Run `(cd sphinx/locale; tx push -s)`
+for first beta releases
+-----------------------
 
-* Update sphinx/locale/<lang>/ files if final major release ...
+* open https://travis-ci.org/sphinx-doc/sphinx/branches and check **master** branch is green
+* Run `git status` and check nothing changed
```

Comment:
This isn't valid rST. Also, you should use `git fetch` first

Reply:
Where is invalid?

I fixed about `git fetch`.

Reply:
Well, not invalid. It's just that the default role has no significance here. That's noted in the docs:

    The default role (`content`) has no special meaning by default. You are free to use it for
    anything you like, e.g. variable names; use the default_role config value to set it to a known
    role.

I think it would be better to format it as a literal with double backticks.

Reply:
You're right. fixed.